libs/common-services/src/main/java/uk/gov/di/ipv/core/library/gpg45/Gpg45Scores.java

```diff
@@ -49,10 +49,7 @@ public Gpg45Scores(Evidence evidence, int activity, int fraud, int verification)
 
```

Comment:
I think we probably should update the other two constructors as well - even if the sorting isn't needed for them!